### PR TITLE
Commit PaystationController.java

### PR DESCRIPTION
**Pull request by navajo00**
PaystationController ready for review. Code Style complies with Google Java Style Guide.

### CODE REVIEW

**Review by duane-mcmahon — APPROVED** [review]
Clean Code.
Intention-Revealing Names
Class is small.
Functions are small.
No duplication of code.
Code formatting is applied:
          Braces
          Block indentation: +2 spaces
          One statement per line
          Column limit: 100
          Line-wrapping
          Class names are written in UpperCamelCase.
          Method names are written in lowerCamelCase.

The features of the code documented above conform to the Google style recommendations for Java.          
In summary, the code for this commit checks out OK (with one formatting defect in relation to use of javadocs: refer comment).

**Comment by WBGroupR** [comment]
As per review, code conforms to Google Java style guide

File: src/bcccp/carpark/paystation/PaystationController.java [hunk 1]
```diff
@@ -12,24 +12,44 @@
   private float charge;
 
   public PaystationController(ICarpark carpark, IPaystationUI ui) {
```
Comment (duane-mcmahon) [inline src/bcccp/carpark/paystation/PaystationController.java:14]: 1. TYPE. Formatting: Google style Infringement
No javadocs present: "At the minimum, Javadoc is present for every public class,
and every public or protected member of such a class".
